apache/rocketmq/v1/service.proto

```diff
@@ -317,13 +315,14 @@ message RecoverOrphanedTransactionRequest {
 }
 
 message MultiplexingRequest {
```

Comment:
After this change, the request is not multiplexed anymore. Should it be renamed to GenericPollingRequest?